colabs/prompts/W&B_LLM_QA_Bot.ipynb

```diff
@@ -0,0 +1,648 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #6.    entity = "wandb" #@param
</pre><p>Explicitly tell user to set their entity to their username from wandb</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/wandb/examples/pull/461/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>